### ISSUE TITLE
using Android Studio 1.1.0 /Gradle System

### ISSUE THREAD

**Issue by sepranagw** [almende/demoapps#1]
I really wanted to see this work, but I think because this demoapp was built not using Android Studio or a non-Gradle system, it's just not compiling.  Tried importing code- guessed at using a 'from Eclipse' import option on Studio, and it said there were multiple dependencies on the original code that had non-gradle dependencies that conflicted with gradle code that Android Studio tried to build around it to make it work.  I made sure that files it didn't import (hidden/not hidden) were included at the top level (pom.xml, lint.xml, project-properties, .idea folder, etc,) but it just couldn't work.  
Would REALLY love to see a Android Studio/Gradle build of this, so this isn't an issue so much as a request...unless I'm missing something?

**Comment by ludost** [comment]
Hi Corinne,

Even more than the few demos in the Eve distribution, these demos were very specific implementations for specific events. Not sure if this will give you a good impression of what Eve can do for you.

The ConferenceApp was a demonstration of combining cloud-based and phone-based agents for supporting face2face meetings between people that share some social network, but don't know each other yet. (e.g. a dating tool, or in this case a conference visitor support tool for networking) The phone-app uses bluetooth signals to find nearby phones and communicates with the agents on those phones to see if the owner is of interest for you. The application was demonstrated during the PAAMS 2014 conference, but hasn't been used since.

Back to your basic interest: We are in the process of defining demos that are better suited for community forming and provide better access to Eve. Can you describe what you would be looking for in a demo? Maybe we can provide you with some more suited example code?

Regards,
Ludo.

**Comment by sepranagw** [comment]
Yeah I suppose the ConferenceApp isn't necessary as a demo to understand the Eve platform.  I'm looking for a multi-agent platform that would be customizable to a hierarchical implementation of decision making which it looks like I can do with Eve.  Getting a hello world type demo that can be started up really quickly just to see basic capabilities is helpful when deciding on an AI platform so that not a whole lot of time is wasted learning how to use the platform well enough to implement something and then discover it's not what's suited to my needs are, is what is ideal.  
MadKit (http://www.madkit.org/) and Jason (http://sourceforge.net/projects/jason/) have really great demos available.  I was able download both of their software bundles and they had a Java GUI from which you could launch demos and see not only the messaging script between agents, but also a visual of what was going on.   They both demonstrated some basic scenarios (Madkit, artificial life platform, multi-player pong; Jason, wumpus world, vacuum cleaner world, etc).  I don't think demos I'd consider have to go this far to provide a GUI interface, I'd just need to be able to launch from command line and see the messaging that happens between agents.  Anything like that, that exists for Eve would be very helpful!
